dialogs/graph_settings.py

```diff
@@ -67,8 +67,24 @@ def __init__(self, parent):
         # Connect and show
         self.OKButton.clicked.connect(self.accept_settings)
         self.cancelButton.clicked.connect(self.close)
-
+        self.radio_range_manual.clicked.connect(lambda: self.toggle_manual(True))
+        self.radio_range_auto.clicked.connect(lambda: self.toggle_manual(False))
+        
+        if self.radio_range_auto.isChecked():
+            self.toggle_manual(False)
+           
+        elif self.radio_range_manual.isChecked():
+            self.toggle_manual(True)
+ 
         self.exec_()
+    
+    def toggle_manual(self, is_manual):
+        # if manual is on, spinboxes are enabled
+        is_disabled = not is_manual
+        self.spin_range_x_min.setDisabled(is_disabled )
+        self.spin_range_x_max.setDisabled(is_disabled )
+        self.spin_range_y_min.setDisabled(is_disabled )
+        self.spin_range_y_max.setDisabled(is_disabled ) 
```

Comment:
```suggestion
        self.spin_range_x_min.setDisabled(is_disabled)
        self.spin_range_x_max.setDisabled(is_disabled)
        self.spin_range_y_min.setDisabled(is_disabled)
        self.spin_range_y_max.setDisabled(is_disabled) 
```
Quick fix to a change I previously suggested.. There should be no whitespace before ")". Sorry about the hassle, writing code in GitHub comment field is not exactly ideal.